### DISCUSSION TITLE
JavaScript Course : Add a new resource in 'Using BaaS For Your Back End' lesson

### DISCUSSION THREAD

**Discussion by Ruchita1010** [TheOdinProject/curriculum#25587] [Ideas]
### Describe your suggestion

I think the [firebase 9 playlist](https://www.youtube.com/playlist?list=PL4cUxeGkcC9jERUGvbudErNCeSZHWUVlb) by Net Ninja will be a great addition to the lesson. It's precise and everything is nicely explained! 
This playlist provides the most up-to-date information on the major changes introduced in Firebase v9.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/javascript-using-baas-for-your-back-end

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Additional Comments

I feel things are better explain in this playlist than in Google Codelab. Codelab was good but sometimes I felt things were just thrown at you, why are we doing it or why are we using this wasn't clear. Just copying and pasting the code to make that feature work. So, I think this playlist should be a part of 'Assignment' and not 'Additional Resource'. Also, the fireship video is great as always but it's quite an old one and doesn't use Firebase v9.

**Comment by KevinMulhern** [comment]
Thanks for the suggestion @Ruchita1010, moving this to a discussion for now.

@TheOdinProject/javascript any thoughts on this?

**Comment by JustWaveThings** [comment]
I'll add that I went through the codelab in the lesson as a learner and wasn't able to get a working version, even after reaching out to Firebase support. In the discord, it seems there are two others in the same boat as me -- @hebi and @twinskylar.   

Here's the thread if it's needed. https://discord.com/channels/505093832157691914/1121497232932339853

**Comment by 01zulfi** [comment]
Closingg since discussion is irelevant now with the [new React course](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new)
